cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -121,7 +127,14 @@ func NewCMServer() *CMServer {
 // of volume.VolumeConfig which are then passed to the appropriate plugin. The ControllerManager binary is the only
 // part of the code which knows what plugins are supported and which CLI flags correspond to each plugin.
 type VolumeConfigFlags struct {
-	PersistentVolumeRecyclerTimeoutNFS int
+	// the pod to use as the default recycler.  default provided by the binary and the filepath flag can override.
+	PersistentVolumeRecyclerPodTemplate *api.Pod
+	// See CLI flag comments in the AddFlags func.
+	PersistentVolumeRecyclerPodTemplateFilePath      string
```

Comment:
I think what you suggest follows the pattern better.  Instead of 1 pod template for all (same image, just timeout override), we'd have the ability to have a custom pod per plugin.  That makes more sense.